realm/src/main/java/io/realm/BaseRealm.java

```diff
@@ -494,8 +509,39 @@ static synchronized void releaseFileReference(RealmConfiguration configuration)
         globalRealmFileReferenceCounter.put(canonicalPath, refCount - 1);
     }
 
+    // Public because of migrations
+    @Deprecated
+    public Table getTable(String className) {
+        className = Table.TABLE_PREFIX + className;
+        Table table = dynamicClassToTable.get(className);
+        if (table == null) {
+            if (!sharedGroupManager.hasTable(className)) {
+                throw new IllegalArgumentException("The type " + className + " doesn't exist in this Realm.");
```

Comment:
This is for the Dynamic interface. I am not sure "class" is the correct word? I have a hard time finding a vocabulary for it. Right now I am using "className" as the variable name, which would suggest I should change the wording like you said, but should we find other words for it?